src/sections/Community/Handbook/projects.js

```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
             <p>
               <h3>
                 <a href="https://layer5.io/">
-                  <img
-                    margin-left="10%"
-                    width="3.5%"
-                    align="left"
-                    src={layer5icon}
-                  />
-                  &nbsp; Layer5{" "}
+                  <img className="project-title-icon" src={layer5icon} alt="Layer5" />
+									&nbsp; Layer5{" "}
                 </a>
               </h3>
             </p>
             <p>
-              Its service mesh community represents the largest collection of
-              service Mesh projects.Emerging projects like GetNighthawk{" "}
-              <img width="2.5%" src={hawkIcon} /> , community with{" "}
-              <img width="2.5%" src={meshmateicon} /> MeshMates, catch-all org,
-              “home base”. <br />
+							Its service mesh community represents the largest collection of
+							service Mesh projects.Emerging projects like GetNighthawk{" "}
+              <img className="project-description-icon" src={hawkIcon} /> , community with{" "}
+              <img className="project-description-icon" src={meshmateicon} /> MeshMates, catch-all
```

Comment:
add a link to the meshmates page here.

Reply:
> I'm sorry to keep doing this @shubham-tyagi sweat_smile, just tiny changes

I don't mind it at all. Every time there is something to learn whenever I see changes required in a pull request, haha. You all have been very patient with my pull requests.